.github/workflows/continuous-integration.yml

```diff
@@ -179,6 +185,9 @@ jobs:
       - name: "Run unit tests with phpunit/phpunit"
         run: vendor/bin/phpunit
 
+      - name: "Check PSR-12 Codestyle"
```

Comment:
Why the random addition of this tool after PHPUnit? Why should it run in the matrix, is PSR12 different for each PHP version and/or Laravel version?

Reply:
it was just a test, deleted it anyway